docs/_data/quotes.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
nice idea, easy to update :)

Reply:
@redonkulus  Where goes the meme I sent you? ;-)

Reply:
lol, that was left on the cutting room floor 🧹